LayoutTests/fast/harness/results.html

```diff
@@ -802,12 +802,9 @@
     layoutTestsBasePath()
     {
         let basePath;
-        if (this.shouldUseTracLinks()) {
-            let revision = this.testResults.revision;
```

Comment:
Not sure if revisions are still attached to test results. I wasn't able to find instances of this so I assume this is a deprecated attribute.

Reply:
Looks like the intention was to link to the exact revision of the test file, as opposed to the latest; but this appears to have broken a long time ago - perhaps with buildbot 2 transition.

Reply:
Do we have a translation service like `https://commits.webkit.org/` but for file/tree views in GitHub?

Reply:
We don't but it wouldn't be particularly difficult to add.